docs/proposals/device-crd.md

```diff
@@ -761,11 +761,11 @@ IoT device lifecycle management comprises of several steps listed below
   - If a device is damaged , it needs to be retired. This is currently not in scope of this design.
 
 ### Device Actions
-- Currently the device model doesn't support [WOT style actions](https://iot.mozilla.org/wot/#actions-resource). The only way to perform some action on the device is by changing the desired state of the twin property in the status field. We need to see how WOT actions can be incorporated in this model and who will consume those actions ?
+- Currently the device model doesn't support [WOT style actions](https://webthings.io/wot/#actions-resource). The only way to perform some action on the device is by changing the desired state of the twin property in the status field. We need to see how WOT actions can be incorporated in this model and who will consume those actions ?
```

Comment:
https://webthings.io/wot/#actions-resource is still a broken link

Reply:
I can't find the correctly link in webthings, any idea about this?

Reply:
> I can't find the correctly link in webthings, any idea about this?
I changed it and it has been merged